staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -328,6 +338,29 @@ func (g *Cloud) ensureInternalFirewall(svc *v1.Service, fwName, fwDesc string, s
 	if err != nil && !isNotFound(err) {
 		return err
 	}
+	// TODO Remove legacyFwName logic after 3 releases, so there would have been atleast 2 master upgrades that would
```

Comment:
TODO needs issue number for tracking

TODO(12345): xxxxxxxxxx

Reply:
Done